packages/Webkul/Product/src/Repositories/ProductCustomerGroupPriceRepository.php

```diff
@@ -33,10 +44,28 @@ public function saveCustomerGroupPrices(array $data, $product)
                     $row['customer_group_id'],
                 ]));
 
+                if (in_array($row['unique_id'], $processedUniqueIds)) {
+                    throw new \Exception(trans('admin::app.catalog.products.edit.price.group.duplicate-error'), self::DUPLICATE_CUSTOMER_GROUP_PRICE);
+                }
+
+                $processedUniqueIds[] = $row['unique_id'];
+
                 if (Str::contains($customerGroupPriceId, 'price_')) {
-                    $this->create(array_merge([
-                        'product_id' => $product->id,
-                    ], $row));
+                    $existingPrice = $this->findOneWhere(['unique_id' => $row['unique_id']]);
+                    if ($existingPrice) {
+                        throw new \Exception(trans('admin::app.catalog.products.edit.price.group.duplicate-error'), self::DUPLICATE_CUSTOMER_GROUP_PRICE);
```

Comment:
The second argument is actually of integer type.  

So, if we are not using or comparing it in the catch block, we can simply remove the second argument and the associated constant as well.  

I initially thought it was being used in the catch block, but if that’s not the case, we can clean it up.